src/main/scala/uk/gov/nationalarchives/api/update/ApiUpdate.scala

```diff
@@ -11,12 +11,11 @@ import scala.concurrent.{ExecutionContext, Future}
 import scala.language.postfixOps
 
```

Comment:
Unnecessary line break

Reply:
I always miss the ones that aren't at the end of the file :smile: